packages/go_router/example/lib/stateful_shell_route.dart

```diff
@@ -60,6 +60,9 @@ class NestedTabNavigationExampleApp extends StatelessWidget {
                 ],
               ),
             ],
+            // To enable preloading of the initial locations of branches, pass
+            // 'true' for the parameter preload.
+            // preload: true,
```

Comment:
I'd find it clearer if the last line was uncommented as `preload: false`

Reply:
You may be right, I'll consider this.